### PR TITLE
add polish language

### PR DESCRIPTION
**Pull request by olekstomek**
add support for polish language

### CODE REVIEW

File: src/i18n.js [hunk 1]
```diff
@@ -208,6 +208,35 @@ const resources = {
...
+  'pl-PL': {
+    translation: {
+      title: 'Karta WiFi',
+      'desc.use':
+        'Wydrukuj prostą kartę z danymi logowania do sieci Wi-Fi. Przyklej go do lodówki, trzymaj w portfelu, itp.',
+      'desc.privacy':
+        'Twoje informacje o Wi-Fi nigdy nie są wysyłane na serwer. Na tej stronie nie stosuje się śledzenia, analiz ani odcisków palców. Zobacz',
+      'desc.source': 'kod źródłowy',
+      'wifi.login': 'Logowanie do WiFi',
+      'wifi.size': 'Rozmiar',
```
Comment (olekstomek) [inline src/i18n.js:220]: it is added because it is used in feature https://github.com/bndw/wifi-card/pull/89

**Comment by olekstomek** [comment]
@bndw I'm guessing my translation hasn't been merged because the branch `bndw:bndw/supported-langs` was previously deleted. Should I do PR to master branch again?

**Comment by bndw** [comment]
@olekstomek !! I'm sorry, I must have unintentionally closed and deleted this branch 🤦 Yes, if you don't mind reopening a PR I'd be happy to have it.

**Comment by olekstomek** [comment]
@bndw Don't worry, everything is fine. I didn't have an idea how to reopen this PR (maybe I can't actually) and that's why here's a new one: https://github.com/bndw/wifi-card/pull/116

**Pull request closed without merging**